pavelib/servers.py

```diff
@@ -28,9 +28,10 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
     if not settings:
         settings = DEFAULT_SETTINGS
 
-    if not skip_assets:
-        # Local dev settings use staticfiles to serve assets, so we can skip the collecstatic step
-        args = [system, '--settings={}'.format(settings), '--skip-collect', '--watch']
+    if asset_settings:
+        args = [system, '--settings={}'.format(asset_settings), '--skip-collect', '--watch']
```

Comment:
My understanding is that ``--skip-collect`` only skips the call to collectstatic but does all of the other asset processing (SASS and Coffee compilation etc). Before my changes, this code assumed that it could always skip collectstatic because the servers didn't run with those assets. However, with this change, we do want to collectstatic if we are running devstack with optimized assets.

Given this, the distinction is that you can disable all asset updating by passing None for ``asset_settings``, while you can skip just the static asset collection by passing ``collect_static=False``. I had attempted to describe this in the docstrings, so let me know if you think it needs more detail.